modules/reference/pages/cluster-properties.adoc

```diff
@@ -293,6 +293,50 @@ Max number of simultaneous connections to S3 per shard. Includes connections use
 
 ---
 
+=== cloud_storage_recovery_topic_validation_mode
+
+Validation performed before recovering a topic from cloud storage. In case of failure, the reason for the failure appears as ERROR lines in the Redpanda application log. For each topic, reports errors for all partitions, but for each partition, only the first error is reported.
+
+This property accepts the following parameters:
+
+- `no_check`: Skips the checks for topic recovery.
+- `check_manifest_existence`:  Runs an existence check on each `partition_manifest`. Fails if there are connection issues to the cloud storage.
+- `check_manifest_and_segment_metadata`: Download the manifest and run a consistency check, comparing the metadata with the cloud storage objects. The process will fail if any metadata references missing cloud storage objects.
```

Comment:
```suggestion
- `check_manifest_and_segment_metadata`: Download the manifest and run a consistency check, comparing the metadata with the cloud storage objects. The process will fail if any metadata references are missing cloud storage objects.
```

Reply:
Not sure if this change is correct. Since the metadata itself don't contain the cloud storage objects. Rather it references the cloud store objects.
The process fails if there's a line in the metadata object saying that a certain object should exist, but it's not there locally.

Reply:
Ah--thanks for that clarification @Deflaimun! I'm deleting the suggestion.